apps/search/templates/search_result.jade

```diff
@@ -1,4 +1,4 @@
-a.search-result( href=result.href(), class=result.get('display_model'), data-id=result.get('id') )
+a.search-result( href=result.href(), class=_s.decapitalize(result.get('display_model')), data-id=result.get('id') )
```

Comment:
Learned `decapitalized`! 👍 

We could also combine that with `dasherize()` if that's friendlier for class names:
```javascript
s("PartnerShow").decapitalize().value()
// "partnerShow"
s("PartnerShow").dasherize().value()
// "-partner-show"
s("PartnerShow").decapitalize().dasherize().value()
// "partner-show"
```